extension/src/popup/views/UnlockAccount/index.tsx

```diff
@@ -41,8 +41,7 @@ export const UnlockAccount = () => {
     const { password } = values;
     await dispatch(confirmPassword(password));
     // skip this location in history, we won't need to come back here after unlocking account
-    history.replace(destination);
-    navigateTo(destination, queryParams);
+    history.replace(`${destination}${queryParams}`);
```

Comment:
fixing a bug where if you weren't logged in when you tried to Sign a transaction and were re-directed to the Unlock Account screen, authenticating was redirecting you to a broken page

@aristidesstaffieri Didn't we talk about this in Slack? I can't seem to find the thread or a Jira ticket :/

Reply:
hmm ya know I don't remember this one specifically but this looks right to me now at a glance